_posts/2018-10-25-meet-n-greet.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: "Meet and Greet"
+slug: meet-and-greet
+text: 'Meet the members of Toronto Mesh'
+location: Free Geek Toronto, 180 Sudbury St
+locationLink: https://osm.org/go/ZX6BI0AGy?node=4042949045
+date: 2018-10-25
+startTime: '19:00'
+endTime: '21:00'
+---
```

Comment:
We should tag this to all our events:

>This event has a [code of conduct](/code-of-conduct/).

Reply:
All events are getting the Code of Conduct line now.